convert_wide_to_long_format/convert_wide_to_long_format.py

```diff
@@ -0,0 +1,65 @@
+import re
+import sys
+import argparse
+import gzip
+
+FINNGENID = 'FINNGENID'
+COLUMN = 'COLUMN';
+VALUE = 'VALUE';
+FILTER_HEADER_KEYWORD = '_FU_'
+
+def print_progress_bar(iteration, total, prefix='', suffix='', length=50):
+    percent = f"{100 * (iteration / float(total)):.1f}"
+    filled_length = int(length * iteration // total)
+    bar = '█' * filled_length + '-' * (length - filled_length)
+    sys.stdout.write(f'\r{prefix} |{bar}| {percent}% {suffix}')
+    sys.stdout.flush()
+    if iteration == total:
+        print()  # Newline at the end
+
+def run(input_file, output_file, columns):
+    with gzip.open(input_file, 'r') as f:
```

Comment:
I don't personally see a progress bar as such an important thing that we want to loop over the file for it. @Fedja any opinions?
Alternative options for progress indicators, if some are wanted, would be the amount of lines processed, and the speed of lines processed per second. But anyways, I usually am not too interested in those.